validators/float_in_slice.go

```diff
@@ -0,0 +1,61 @@
+package validators
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+const (
+	floatInSliceValidatorErr = "number must be one of [%s]"
+)
+
+type floatInSliceValidator struct {
+	values   []float64
+	errormsg string
+}
+
+func floatInSlice(values ...float64) tfsdk.AttributeValidator {
+	return floatInSliceValidator{
+		values:   values,
+		errormsg: strings.Trim(strings.Replace(fmt.Sprint(values), " ", ", ", -1), "[]"),
+	}
+}
+
+func (v floatInSliceValidator) Description(context.Context) string {
+	return fmt.Sprintf(floatInSliceValidatorErr, v.errormsg)
```

Comment:
Could we make this a description instead of an error message? See an example [here](https://github.com/frankgreco/terraform-helpers/blob/master/validators/compare.go#L16).

Reply:
Hmm... I followed the pattern from `string_in_slice.go`. Can you check once again?

Reply:
That was a copy/paste error when I created `string_in_slice`. Hence why I included a link ([here](https://github.com/frankgreco/terraform-helpers/blob/master/validators/compare.go#L16)) to a correct example.